src/meetup_manager/clients/github.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+from typing import List
+
+from django.conf import settings
+from django.core.cache import cache
+from github import Github, GithubException
+from github.Label import Label
+from github.PaginatedList import PaginatedList
+
+from .base import BaseIntegrationClient
+
+ACCESS_TOKEN_KEY = "GITHUB_ACCESS_TOKEN"
+
+logger = logging.getLogger(__name__)
+TALK_PROPOSAL = "talk-proposal"
+
+
+class GithubClient(BaseIntegrationClient):
+    """
+    Integration client for github interactions using Github V3 API through
+    PyGithub (https://pygithub.readthedocs.io).
+    """
+    api_endpoint = "https://api.github.com"
+
+    @staticmethod
+    def access_token():
+        """
+        Return access token configured from cache. Set token to cache if not
+        set previously.
+        Access token should be configured through environment variables
+        return: Github access token configured.
+        """
+        access_token = cache.get(
+            ACCESS_TOKEN_KEY, default=os.getenv(ACCESS_TOKEN_KEY, "")
+        )
+        cache.set(ACCESS_TOKEN_KEY, access_token)
+        return access_token
+
+    def __init__(self):
+        super(GithubClient, self).__init__()
+        self.client = Github(GithubClient.access_token())
+
+    def notify_speaker(self, issue_id: int, comment: str):
```

Comment:
@GoelJatin & @sarathsomana - What is the idea behind notifying speakers?

Reply:
Wrt to github integration, its about adding a comment on the issue about proposal status / questions etc., 
These notifications can be directly managed from admin interface

Reply:
We were thinking of how we can notify the speakers as well that their talk has been scheduled as an official notification.

So thought we can just add a comment on the Talk proposal itself, so that

a. it is also public
b. we don't have to build our own notification system for this and let GitHub send the email whenever there's an activity on the issue

Reply:
Ok, a couple of points from my end. If we send a custom email to the speaker confirming the availability etc, that will be great. So far, we have been selecting talks by adding a label called "scheduled". So that can be done for talk selection as well.